client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -103,6 +103,16 @@
      */
     CompletableFuture<Boolean> deleteScope(final String scopeName);
 
+    /**
+     * API to delete a scope recursively. This method once invoked will not allow any
+     * operation(create/update/seal/delete) on Stream/Rg/KVT within the scope
```

Comment:
This is not true. We do allow create, update, seal and delete APIs to be invoked on the scope or its children after deleteScopeRecursive has been invoked. Please remove this line from the javadoc.

Reply:
The javadoc has been updated as per suggestion.